hexa/workspaces/schema.py

```diff
@@ -73,6 +74,12 @@ def resolve_workspace_permissions(workspace: Workspace, info):
     return workspace
 
 
+@workspace_object.field("database")
+def resolve_workspace_database(workspace: Workspace, info, **kwargs):
+    setattr(workspace, "database", Database.objects.get(database="hexa-explore-demo"))
```

Comment:
No need for this, it's a bit confusing.

Reply:
Better to do a database = Database.objects.get(database="hexa-explore-demo")) in each util functions